### DISCUSSION TITLE
Make osu!lazer a LOT more customizable

### DISCUSSION THREAD

**Discussion by axurright** [ppy/osu#28266] [Ideas]
I think osu!lazer can be made more customizable, to the point were you can edit most of the game client, including the color amd font of the ENTIRE gui

**Comment by Stoppedpuma** [comment]
If you mean skinning, you can probably find what you're looking for [here](https://github.com/ppy/osu/discussions?discussions_q=is%3Aopen+skin+editor).

**Comment by axurright** [comment]
No, what I mean is that by default you can customize the entire client, even the GUI

**Reply by Stoppedpuma** [reply]
So things like moving the back button or changing the colour of menus, etc? This falls under skinning. See the link above and look through the various discussions.

**Comment by murryguy** [comment]
> I think osu!lazer can be made more customizable, to the point were you can edit most of the game client, including the color amd font of the ENTIRE gui

Do you mean "and"

**Reply by axurright** [reply]
Yes
